Carrot-Assistant/components/embeddings.py

```diff
@@ -30,6 +42,22 @@ def __init__(
             model: EmbeddingModel,
             search_kwargs: dict,
             ) -> None:
+        """
+        Initialises the connection to an embeddings database
+
+        Parameters
+        ----------
+        embeddings_path: str
+            A path for the embeddings database. If one is not found, it will be built, which takes a long time. This is built from concepts fetched from the OMOP database
+        force_rebuild: bool
+            If true, the embeddings database will be rebuilt.
+        embed_vocab: List[str]
+            A list of OMOP vocabulary_ids. If the embeddings database is built, these will be the vocabularies used in the OMOP query
+        model: EmbeddingModel
+            The model used to create embeddings
```

Comment:
can we expand this here? which model could this be? I know that there is a list, but we could hint about the options?